src/components/ProfilePage/QuestsOverview/QuestsOverview.tsx

```diff
@@ -0,0 +1,94 @@
+import { useTranslation } from 'react-i18next';
+import { useOngoingQuests } from 'src/hooks/useOngoingQuests';
+import type { Quest } from 'src/types/loyaltyPass';
+import { checkInclusion } from '../checkInclusion';
+import { QuestCard } from '../QuestCard/QuestCard';
+import { QuestCarouselNumericItems } from './QuestCarouselNumericItems';
+import {
+  QuestsOverviewContainer,
+  QuestsOverviewTitle,
+} from './QuestsOverview.style';
+
+interface QuestCarouselProps {
+  pastCampaigns?: string[];
+  traits?: string[];
+  // traits?: Trait[];
+}
+
+export const QuestsOverview = ({
+  pastCampaigns,
+  traits,
+}: QuestCarouselProps) => {
+  const { t } = useTranslation();
+  const { quests, isLoading: isQuestsLoading, url } = useOngoingQuests();
+
+  return (
+    <QuestsOverviewContainer>
+      <QuestsOverviewTitle variant="headerMedium">
+        {t('missions.available')}
+      </QuestsOverviewTitle>
+      {/** render quests */}
+      {!isQuestsLoading
+        ? quests?.map((quest: Quest, index: number) => {
+            const baseURL = quest.attributes.Image?.data?.attributes?.url;
+            const imgURL = new URL(baseURL, url.origin);
+            const rewards = quest.attributes.CustomInformation?.['rewards'];
+            const rewardType =
+              quest.attributes?.CustomInformation?.['rewardType'];
+            const rewardRange =
+              quest.attributes?.CustomInformation?.['rewardRange'];
+            const chains = quest.attributes.CustomInformation?.['chains'];
+            const claimingIds =
+              quest.attributes?.CustomInformation?.['claimingIds'];
+            const rewardsIds =
+              quest.attributes?.CustomInformation?.['rewardsIds'];
+            const questTraits = quest.attributes?.CustomInformation?.['traits'];
+            const endDate = quest.attributes.EndDate;
```

Comment:
What if quest is null?

Reply:
Should not be?
![image](https://github.com/user-attachments/assets/41974dff-4bd5-443e-b4fa-a0d1f98eed3b)

Reply:
Might not because we do not know if it is a success or not there, we only check if it has been loaded

Reply:
@oktapodia  Should we check on `"success"-state `instead? Or for `"quests?.length"` ? Or "`!isQuestsLoading && isQuestsSuccess"`

Reply:
The thing is that, it is not consistent, either we have `quest.attributes` everywhere or `quest?.attributes`